Makefile

```diff
@@ -161,7 +161,7 @@ $(GOLINT):
 
 .PHONY: check
 check: $(GOLINT)
```

Comment:
This does depend on `gofmt` too, right? We should add a target for getting `gofmt` at some moment...